docs/src/howto.md

```diff
@@ -0,0 +1,123 @@
+# How To?
+
+This is a list of frequently ask questions answering basic "How to do X, Y, Z?" questions.
+
+For all the questions below you need to start with:
+
+```julia
+using Javis
+
+function ground(args...)
+    background("black")
+    sethue("white")
+end
+
+video = Video(500, 500)
+
+javis(video, [
+    BackgroundAction(1:100, ground),
+    SNIPPETS_GO_HERE
+]; pathname="how_to.gif")
+```
+
+Each of the code snippets should be part of `SNIPPETS_GO_HERE`.
+
+## How can I move a circle from A to B?
+
+First of all you need to define an [`Action`](@ref) which draws a circle.
+
+```julia
+Action(1:100, (args...)->circle(O, 50, :fill))
+```
+
+and then you need the [`Translation`](@ref) command to move the circle.
+
+```julia
+Action(1:100, (args...)->circle(O, 50, :fill), Translation(O, Point(100, 100))
+```
+
+The circle then moves from the origin (center of frame) 100 px down and to the right.
+
+## How can I define frames?
+
+There are currently three different ways to define frames inside Javis.
+The simplest one is to define the `UnitRange` like `1:100` as above such that the action is called for every frame from `1` to `100`.
+
+**Examples:**
+```julia
+Action(1:100, (args...)->circle(O, 50, :fill)),
+Action(1:50, (args...)->circle(O, 20, :stroke))
+```
```

Comment:
Maybe explain how this first action creates a ball at the origin of size 50 for the first hundred frames and the second action creates a ball the size of 20 for the first 20 frames and further explain that the balls would be on top of each other.

Reply:
I increased the radius of the second one. Thanks for pointing it out. I think in the "how to" we should only answer the question asked. At that point I assume they read at least the first tutorial and know the basics.